.travis.yml

```diff
@@ -3,11 +3,11 @@ sudo: false
 matrix:
     include:
     - python: "3.5"
-      env: DEPS="numpy=1.11.2 scipy=0.18.1 cython nose matplotlib"
+      env: DEPS="numpy=1.11.2 scipy=0.18.1 cython nose matplotlib scikit-learn=0.18.1"
     - python: "3.4"                                                                                         
-      env: DEPS="numpy=1.10.4 scipy=0.17 cython nose matplotlib"
+      env: DEPS="numpy=1.11.2 scipy=0.17 cython nose matplotlib scikit-learn=0.18.1"
```

Comment:
why would you do that? Why not upgrade numpy? I always use a `virtualenv` for python, even on a supercomputer. Any parrticular reason why you would not upgrade from numpy 1.10.4?